src/main/java/Task.java

```diff
@@ -0,0 +1,41 @@
+public abstract class Task {
+    private final String description;
+    protected enum Status {
+        DONE,NOT_DONE,URGENT
+    }
+    private Status status;
+
+    /** Private constructor for a Task */
+    public Task(String description) {
+        this.description = description;
+        this.status = Status.NOT_DONE;
+    }
+    protected void updateStatus(Task.Status input) {
+        StringBuilder changelog = new StringBuilder();
```

Comment:
Similar issue here, this name does not really sound like a string, maybe rename it changeLogToString would make it more intuitive for everyone